### PR TITLE
Add test cases for the skip login consent property

### PR DESCRIPTION
**Pull request by DMHP**
Refactored existing OIDC test cases.
Added two new test cases.
 - OIDCSPWiseSkipLoginConsentTestCase
- OIDCFileBasedSkipLoginConsentTestCase

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSPWiseSkipLoginConsentTestCase.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        ServiceProvider serviceProvider = appMgtclient.getApplication(oidcApplication.getApplicationName());
+        LocalAndOutboundAuthenticationConfig localAndOutboundAuthenticationConfig =
+                new LocalAndOutboundAuthenticationConfig();
+        localAndOutboundAuthenticationConfig.setSkipConsent(true);
+        serviceProvider.setLocalAndOutBoundAuthenticationConfig(localAndOutboundAuthenticationConfig);
+        appMgtclient.updateApplicationData(serviceProvider);
+    }
+
+    @Test(groups = "wso2.is", description = "Test authz endpoint before creating a valid session")
+    public void createUserSession() throws Exception {
```
Comment (ShanChathusanda93) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSPWiseSkipLoginConsentTestCase.java]: Shall we rename the test method as "testCreateUserSession" as this is a test method.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        user.setProfile(profile);
+        user.addUserClaim(emailClaimUri, email);
+        user.addUserClaim(firstNameClaimUri, firstName);
+        user.addUserClaim(lastNameClaimUri, lastName);
+        user.addRole(role);
+    }
+
+    /**
+     * Intiates an application
+     *
```
Comment (ShanChathusanda93) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCUtilTest.java]: As my view this " * " is an extra one.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAuthCodeGrantSSOTestCase.java [hunk 3]
```diff
@@ -387,34 +337,34 @@ private void testUserClaims() throws Exception {
...
-        playgroundApp.addRequiredClaim(lastNameClaimUri);
-        applications.put(playgroundAppOneAppName, playgroundApp);
-
-        playgroundApp = new OIDCApplication(playgroundAppTwoAppName, playgroundAppTwoAppContext,
-                playgroundAppTwoAppCallBackUri);
-        playgroundApp.addRequiredClaim(emailClaimUri);
-        playgroundApp.addRequiredClaim(firstNameClaimUri);
-        playgroundApp.addRequiredClaim(lastNameClaimUri);
-        applications.put(playgroundAppTwoAppName, playgroundApp);
+        OIDCApplication playgroundApp = new OIDCApplication(OIDCUtilTest.playgroundAppOneAppName, OIDCUtilTest.playgroundAppOneAppContext,
```
Comment (sachiniWettasinghe) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAuthCodeGrantSSOTestCase.java]: this line seems to be exceeding 120 characters

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSPWiseSkipLoginConsentTestCase.java [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    }
+
+    @Test(groups = "wso2.is", description = "Initiate authentication request from playground.apptwo")
+    public void testIntiateLoginRequestForAlreadyLoggedUser() throws Exception {
+
+        testSendAuthenticationRequest(OIDCUtilTest.applications.get(OIDCUtilTest.playgroundAppTwoAppName), false);
+    }
+
+    private void testSendAuthenticationRequest(OIDCApplication application, boolean isFirstAuthenticationRequest)
+            throws Exception {
```
Comment (sachiniWettasinghe) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSPWiseSkipLoginConsentTestCase.java]: This method seems to be duplicated in OIDCFileBasedSkipLoginConsentTestCase as well. Can we get this to the OIDCAbstractIntegrationTest class?

**Pull request merged**